### DISCUSSION TITLE
ONTAP: Security metrics are empty

### DISCUSSION THREAD

**Discussion by twodotoh** [NetApp/harvest#2639] [Q&A]
Just deployed the latest Harvest 23.11 and dont seem to have any metrics being pulled through for the ONTAP: Security dashboard.  Might of been like this before but I also notice that the datacenter variable drop down in the dashboard is empty for some reason, should be 3 listed in there at least.

**Comment by cgrinds** [comment]
hi @twodotoh how did you install Harvest? Can you send you logs based on your install method outline https://netapp.github.io/harvest/nightly/help/log-collection/ to ng-harvest-files@netapp.com

**Reply by twodotoh** [reply]
Using NAbox and dropped the tar.gz into the upgrade, looks to applied everything else

**Reply by cgrinds** [reply]
Thanks. You can collect the log files by following https://netapp.github.io/harvest/nightly/help/log-collection/#nabox

Can you ssh into nabox, run `dc ps -a` and paste the results?

**Reply by twodotoh** [reply]
I dont think I reset the dashboards in NABOX after updating Harvest, is this still a requirement for NABOX based installations of Grafana? could that be the issue?

**Reply by rahulguptajss** [reply]
After upgrading Harvest in NABox, always perform a Reset to synchronize dashboards for the new version. If problems persist after the reset, please share the logs.

**Reply by twodotoh** [reply]
Thanks Rahul, can I assume my alerting rules setup which ties to my dashboards will still be preserved when I do this?

**Reply by rahulguptajss** [reply]
Reset button only overwrites Harvest Grafana dashboards. Alerting rules setup will be preserved.
